internal/services/netapp/netapp_volume_resource_test.go

```diff
@@ -302,6 +302,7 @@ resource "azurerm_netapp_volume" "test" {
   service_level       = "Standard"
   subnet_id           = azurerm_subnet.test.id
   storage_quota_in_gb = 100
+  throughput_in_mibps = 1.562
```

Comment:
this seems odd given its optional computed - how come the test needs this?

Reply:
@katbyte , when "qos_type" of capacity pool is "Manual", it requires throughput_in_mibps of netapp volume is set otherwise it would throw below error message.

Error:
```
polling after CreateOrUpdate: Code="BadRequest" Message="An invalid value '' was received for Volume.ThroughputMibps. Must be 1 or higher" Details=[{"code":"InvalidValueReceived","message":"An invalid value '' was received for Volume.ThroughputMibps. Must be 1 or higher"}]
```